### ISSUE TITLE
Contributor sees the Token balance for each Token listed in the Token tab for the Address page

### ISSUE THREAD

**Issue by gfreh** [poanetwork/blockscout#564]
As a `Contributor`, I want to see a `Token` tab in the `Address Details` page, where I can see the list of `Tokens` the `Address` holds, as well as see the `Token Transfers` for each of those `Tokens`.

https://projects.invisionapp.com/share/8EMX9IOR2V9#/screens/308371747

This continues the work on #410.

## Acceptance Criteria

- For each `Token`, there should be possible to see the following information:
    - `Token` balance (for the associated address) followed by the token symbol.
        - For `ERC20 Tokens`, the balance should be formatted as currency (thousands and decimal separators) considering the token's `decimals` info (if there is no `decimals` info, display the formatted amount). Example: `999,000.02 FREH`
        - For `ERC721 Tokens`, the balance of non-fungible tokens. Example: `803,843 CK`
        - In case the `Token` has no `symbol`, show nothing next to the amount.
    -  If `balance = 0`, the `Token` should be at the bottom of the list.

### Out of Scope
- This issue doesn't include the search field as shown in the mockup.
- This issue doesn't include the token icons as shown in the mockup.
- This issue doesn't include the token balance conversion to USD, which includes:
  - the USD exchange rate for that specific token
  - how much USD that amount of token is worth

### Dependencies
- [x] Depends on #514.

### Task breakdown
- [x] Change query that gets the tokens an address holds to include the token balance.
  - [x] Considers the last block.
- [x] Make the balance available in the Token struct, like the number of transfers.*
- [x] Update the front-end to include the balance.
  - [x] Format just Tokens that have the type ERC-20 (use `CurrencyHelper`)
  - [x] Consider symbol at the end of the value. Ex. (100 Freh)

**Comment by feliperenan** [comment]
@acravenho which will be the order criteria? Today we are ordering by type and name, but it may be necessary to order by token balances.
